### PR TITLE
perf(ui/pull-refresh): add class

### PR DESCRIPTION
**Pull request by wangKBweb**
## Checklist

---

- [ ] Fix linting errors
- [ ] Tests have been added / updated (or snapshots)

## Change information

---

### CODE REVIEW

File: packages/varlet-ui/src/popup/Popup.tsx [hunk 1]
```diff
@@ -45,7 +47,7 @@ export default defineComponent({
 
       return (
         <div
-          class={['var-popup__overlay', overlayClass]}
+          class={[classes(n('overlay')), overlayClass]}
```
Comment (BeADre) [inline packages/varlet-ui/src/popup/Popup.tsx:50]: 这地方直接都用classes包起来吧

File: packages/varlet-ui/src/pull-refresh/PullRefresh.vue [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+    :class="classes(n())"
     @touchstart="touchStart"
     @touchmove="touchMove"
     @touchend="touchEnd"
     @touchcancel="touchEnd"
   >
     <div
-      class="var-pull-refresh__control var-elevation--2"
-      :class="[isSuccess ? 'var-pull-refresh__control-success' : null]"
+      :class="classes(n('control'), 'var-elevation--2', isSuccess ? n('control-success') : '')"
```
Comment (BeADre) [inline packages/varlet-ui/src/pull-refresh/PullRefresh.vue:11]: 条件判断用数组

File: packages/varlet-ui/src/pull-refresh/PullRefresh.vue [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
       :style="controlStyle"
     >
-      <var-icon :name="iconName" :transition="200" :class="iconClass" var-pull-refresh-cover />
+      <var-icon
+        :name="iconName"
+        :transition="200"
+        :class="classes(n('icon'), refreshStatus === 'loading' ? n('animation') : '')"
```
Comment (BeADre) [inline packages/varlet-ui/src/pull-refresh/PullRefresh.vue:17]: 这里和上面一样

Comment (wangKBweb) [reply packages/varlet-ui/src/pull-refresh/PullRefresh.vue]: 改好惹

File: packages/varlet-ui/src/popup/Popup.tsx (hunk 1 above)
Comment (wangKBweb) [reply packages/varlet-ui/src/popup/Popup.tsx:50]: 包起来报错了浩哥

**Review by BeADre — APPROVED** [review]

**Pull request merged**